kolibri/plugins/learn/assets/src/composables/useDownloadRequests.js

```diff
@@ -102,32 +65,36 @@ export default function useDownloadRequests(store) {
     return { loading, storageInfo };
   }
 
+  // add the page route here
   function navigateToDownloads() {}
 
   function addDownloadRequest(content) {
-    const resource_metadata = {
+    const metadata = {
       title: content.title,
       file_size: content.files.reduce((size, f) => size + f.file_size, 0),
       learning_activities: content.learning_activities,
     };
-    const requestData = {
-      node_id: content.id,
-      resource_metadata,
-      user_id: store.getters.currentUserId,
-      reason: 'USER_INITIATED',
-      facility_id: store.getters.userFacilityId,
-      status: 'QUEUED',
-      date_added: new Date(),
-    };
+
+    client({
```

Comment:
You have defined APIResource objects above, but then are not using them here - might be easier to use them?